### DISCUSSION TITLE
Tuning with embedded hardware in the loop?

### DISCUSSION THREAD

**Discussion by settle** [keras-team/autokeras#1606] [Q&A]
@haifeng-jin and others, If I missed this discussed already somewhere here my apologies, but does autokeras have a mechanism for tuning throughput, latency, etc, with hardware in the loop, say for an embedded device that itself cannot do training, and if so are there any examples?

**Comment by settle** [comment]
@haifeng-jin and @qingquansong Circling back in case one of you could still answer this question of mine regarding support for embedded hardware in the loop, thanks!

**Reply by haifeng-jin** [reply]
We are considered this issue pretty important. However, we don't have a good solution for it yet.

**Reply by settle** [reply]
@haifeng-jin Thank you for your reply.  Would you mind sharing the current mechanism by which it is/could be supported, even if it's not an ideal solution yet?
